src/proto/mgmt/srv.proto

```diff
@@ -131,3 +137,20 @@ message ScmModule {
 	// The firmware revision of the module.
 	//string fwrev = 10;
 }
+
+// Identifier for server rank within DAOS pool
+message DaosRank {
+	string pool_uuid = 1;	// UUID of the pool
+	uint32 rank = 2;	// Server rank
+}
+
+enum DaosRequestStatus {
```

Comment:
I don't think we've ever had a conversation on how we wanted to handle module error codes. If they were to be defined on the DRPC level or if they would be nested in the body of the Response structure.  I understand these are for the grpc service handlers between dmg and daos_server but I would imagine you need to use ERR_INVALID_RANK within a drpc call as well and we now have 2 conflicting errno spaces. Perhaps @kjacque @tanabarr and @dpquigl should arrange to figure this out.

Reply:
https://github.com/daos-stack/daos/pull/253/files is the follow-on patch, I didn't think that would be conflicting as it specific to the message PROTO__DAOS_REQUEST_STATUS__ERR_INVALID_RANK

Reply:
In my view the module error codes should be nested in the body. I think at the dRPC level the code is about how the message was processed, whether a handler was found, things like that.

Reply:
yes that's the way I've been proceeding